### ISSUE TITLE
activation code

### ISSUE THREAD

**Issue by bowotulus** [movoleg/aqua-controller#3]
how to get activation code?

**Comment by movoleg** [comment]
It costs $25

**Comment by PavlegG** [comment]
Hey! I want an activation key!) write to me at nerosis@gmail.com and discuss how to transfer money)

**Comment by kandiki** [comment]
> It costs $25

I would like to buy. how do i proceed with it?

**Comment by movoleg** [comment]
Do you have a crypto? Because I don't have other ways to get money. If yes, then you pay me, you download and upload to ESP32 then I generate an activation code for you.

**Comment by kandiki** [comment]
Yes i have setup the code on esp32. just need the activation code. let me know an email id or number to pvt crypto details
